source/openapi-data-api-v1.yaml

```diff
@@ -0,0 +1,991 @@
+openapi: 3.1.0
+info:
+  version: 1.0.0
+  title: MongoDB Atlas Data API
+  description: An API to read, write, and aggregate data in MongoDB Atlas
+security:
+  - AccessToken: []
+  - Email: []
+    Password: []
+  - ApiKey: []
+  - CustomJwt: []
+servers:
+  - description: Data API Base URL
+    url: https://{region}.mongodb-api.com/app/{appId}/endpoint/data/v1
+    variables:
+      appId:
+        default: "your-app-id"
+      region:
+        default: "data"
+        enum:
+          # Global Deployment
+          - "data"
+          # AWS Local Deployment
+          - "us-east-1.aws.data"
+          - "us-west-2.aws.data"
+          - "us-east-2.aws.data"
+          - "eu-central-1.aws.data"
+          - "eu-west-1.aws.data"
+          - "eu-west-2.aws.data"
+          - "ap-southeast-1.aws.data"
+          - "ap-southeast-2.aws.data"
+          - "ap-south-1.aws.data"
+          - "sa-east-1.aws.data"
+          # Azure Local Deployment
+          - "eastus2.azure.data"
+          - "westus.azure.data"
+          - "westeurope.azure.data"
+          - "eastasia.azure.data"
+          - "southeastasia.azure.data"
+          # GCP Local Deployment
+          - "us-central1.gcp.data"
+          - "us-east4.gcp.data"
+          - "us-west1.gcp.data"
+          - "europe-west1.gcp.data"
+          - "asia-south1.gcp.data"
```

Comment:
i worry this could get out of date. could we just make the data type string, have a good description w a few examples, including for global apps, and link to the list of these elsewhere in the docs?